spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/write/BigQueryWriteHelper.java

```diff
@@ -128,16 +178,18 @@ void loadDataToBigQuery() throws IOException {
     List<String> optimizedSourceUris = SparkBigQueryUtil.optimizeLoadUriListForSpark(sourceUris);
     JobInfo.WriteDisposition writeDisposition =
         SparkBigQueryUtil.saveModeToWriteDisposition(saveMode);
-    Schema schema =
```

Comment:
Why was this removed?

Reply:
This was moved to the writeDataFrameToBigQuery() function to use the schema to create temp table